src/components/interaction-designer/block-editors/GenericContactPropertyEditor.vue

```diff
@@ -232,6 +232,13 @@ export class GenericContactPropertyEditor extends mixins(Lang) {
     return !isEmpty(this.subscriberPropertyFields)
   }
 
+  get subscriberPropertyFieldsForSelector() {
```

Comment:
let's try to give it a return type if possible
`IContactPropertyOption & {$isDisabled: boolean}`?

Reply:
Agree & added !